cb-tools/burgbox/+file/modDate.m

```diff
@@ -1,13 +1,18 @@
 function d = modDate(p)
-%UNTITLED Summary of this function goes here
-%   Detailed explanation goes here
+%FILE.MODDATE Returns date modified of files and folders
+%  Returns datenums of files and folders contained in input path(s), p
+%  Input:
```

Comment:
hate to be this guy, but can we be consistent with our docstrings? I'm fine with putting types in parentheses, but we should do this consistently. (Personally, I also prefer the `|` as compared to the `or` but not a huge deal). Also, can we use `:` instead of `-` as previously discussed, and `` ` `` `` ` `` around the variable names? And always add an example function call?